dll/win32/newdev/wizard.c

```diff
@@ -579,6 +579,38 @@ AutoDriver(HWND Dlg, BOOL Enabled)
     IncludePath(Dlg, Enabled & IsDlgButtonChecked(Dlg, IDC_CHECK_PATH));
 }
 
+static INT CALLBACK
+BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lParam, LPARAM lpData)
+{
+    BOOL bValid = FALSE;
+
+    switch (uMsg)
+    {
+        case BFFM_INITIALIZED:
+        {
+            PCWSTR pszPath = ((PDEVINSTDATA)lpData)->CustomSearchPath;
+
+            bValid = CheckBestDriver((PDEVINSTDATA)lpData, pszPath);
+            SendMessageW(hwnd, BFFM_SETSELECTION, TRUE, (LPARAM)pszPath);
+            SendMessageW(hwnd, BFFM_ENABLEOK, 0, bValid);
+            break;
+        }
+
+        case BFFM_SELCHANGED:
+        {
+            WCHAR szDir[MAX_PATH];
+
+            if (SHGetPathFromIDListW((LPITEMIDLIST)lParam, szDir))
+            {
+                bValid = CheckBestDriver((PDEVINSTDATA)lpData, szDir);
+            }
+            SendMessageW(hwnd, BFFM_ENABLEOK, 0, bValid);
```

Comment:
I have tested, `SendMessageW` doesn't work here. But `PostMessageW` will work.
```suggestion
            PostMessageW(hwnd, BFFM_ENABLEOK, 0, bValid);
```